guide/14-deep-learning/how-named-entity-recognition-works.ipynb

```diff
@@ -108,8 +108,8 @@
     "        - labels.csv: 'B-Person', 'I-Person', 'I-Person', 'L-Person', 'O', 'O', 'O', 'U-Country'\n",
```

Comment:
<p><code>spacy.displacy.render(doc,jupyter=True, style='ent',options=options)</code></p><p> </p><p>Suggestion: display spelt wrong. Should it be changed to the following:</p><p> </p><p><code>spacy.display.render(doc,jupyter=True, style='ent',options=options)</code></p><p> </p><p>Kindly check.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/644/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>